### ISSUE TITLE
Freezes on world load/creation.

### ISSUE THREAD

**Issue by K0-RR** [polgaria/netherHigher#15]
Was working fine, I'm not if an update to another mod broke it or what.
https://paste.ee/p/CY9Ni

**Comment by geniiii** [comment]
I'm not seeing anything that would indicate netherHigher is at fault. Does it not freeze without it?

**Comment by K0-RR** [comment]
Yes, If I remove this mod I can load world just fine. I'm using other mods too so maybe something conflicts.

**Comment by geniiii** [comment]
> Yes, If I remove this mod I can load world just fine. I'm using other mods too so maybe something conflicts.

Odd that nothing shows up in the logs, I'll investigate when I can.
